### ISSUE TITLE
Term Request: Monocular diplopia

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11735]
### Term Request: Monocular diplopia

**Preferred term label:**
Monocular diplopia

**Synonyms:**
* Monocular double vision
* Unocular diplopia
* Monocular ghosting

**Definition (free text):**
A visual perception of two images of a single object seen when viewing with only one eye, which persists upon occluding the contralateral eye. Unlike binocular diplopia (which is caused by ocular misalignment and resolves with monocular occlusion), monocular diplopia is typically caused by optical aberrations in the eye's media (such as irregular astigmatism, corneal ectasia, lens opacity/cataract, or subluxation) or rarely by retinal or retrochiasmal cortical pathology.

**Parent term (use hpo.jax.org/app):**
Abnormality of vision (HP:0000504)

**Your nano-attribution (ORCID):**
0000-0001-9042-0508
